File: app/src/main/java/com/bq/zowi/models/ZowiName.java
package com.bq.zowi.models;

/* JADX INFO: loaded from: classes.dex */
public class ZowiName {
    private static final String FACTORY_ZOWI_NAME = "#";
    private String name;

    public ZowiName(String name) {
        this.name = name;
    }

    public String toString() {
        return this.name;
    }

    public static boolean isFactoryName(String name) {
        return name.equals(getFactoryName());
    }

    public static String getFactoryName() {
        return FACTORY_ZOWI_NAME;
    }
}

File: zowi-core/src/main/kotlin/com/bq/zowi/models/ZowiName.kt
package com.bq.zowi.models

class ZowiName(private val name: String) {

    override fun toString(): String = name

    companion object {
        private const val FACTORY_ZOWI_NAME = "#"

        @JvmStatic
        fun isFactoryName(name: String): Boolean = name == FACTORY_ZOWI_NAME

        @JvmStatic
        fun getFactoryName(): String = FACTORY_ZOWI_NAME
    }
}
